.gitignore

```diff
@@ -1,4 +1,3 @@
-config.yml
```

Comment:
Does this have any impact on SaaS provisioning, if there will already be a `config.yml`, where there mightn't have been before?